handler/metadata.go

```
@@ -141,6 +144,36 @@ func (h *XlsxCreate) AddMetaDataToExcelStructure(ctx context.Context, excelInMem
 
 	processMetaElement("Version", strconv.Itoa(meta.Version.Version), true)
 	processMetaElement("Dataset version", meta.DatasetLinks.LatestVersion.URL, true)
+	processMetaElement("Statistical Disclosure Control Statement", sdcStatement, true)
+	processMetaElement("Area Type", areaTypeStatic, true)
+
+	for _, dimensions := range meta.Version.Dimensions {
+		if dimensions.IsAreaType == &trueVal {
+			processMetaElement("Area Type Name", dimensions.Label, true)
+			processMetaElement("Area Type Description", dimensions.Description, true)
+		}
+
+		if dimensions.IsAreaType == &falseVal {
+			processMetaElement("Variable Name", dimensions.Label, true)
+		}
+
+	}
+
+	processMetaElement("Coverage", coverageStatic, true)
+
+	if meta.DatasetDetails.VersionsList.Items != nil {
+		formatToParse := "2006-01-02T15:04:05.000Z"
+		rowNumber++
+		processMetaElement("Version History", "", false)
+		for _, versions := range meta.DatasetDetails.VersionsList.Items {
+			processMetaElement("Version Number", strconv.Itoa(versions.Version), true)
+			date, _ := time.Parse(formatToParse, versions.ReleaseDate)
```

better not to ignore error and add a condition as in case of error as default date will be returned in case of error